### ISSUE TITLE
VoiceOver doesn't read expansion state of headings

### ISSUE THREAD

**Issue by PeterSaathoff-Harshfield** [Ranchero-Software/NetNewsWire#2242]
Environment:
iOS 13.5.1
NNW 5.0.2

Steps:
1. Turn on VoiceOver
2. Navigate VoiceOver's focus to the "Smart feeds" or "On my iPhone" heading

Expected result:
VoiceOver should announce whether the section is expanded or collapsed

Actual result:
VoiceOver reads "Smart feeds, heading" for example. Double-tapping to expand and collapse the section works, but VoiceOver doesn't indicate whether it's expanded or collapsed. For Sighted users, the disclosure triangle serves this purpose.
